python/private/hermetic_runtime_repo_setup.bzl

```diff
@@ -109,7 +109,26 @@ def define_hermetic_runtime_toolchain_impl(
     cc_library(
         name = "python_headers",
         deps = select({
-            "@bazel_tools//src/conditions:windows": [":interface", ":abi3_interface"],
+            "@bazel_tools//src/conditions:windows": [":interface"],
+            "//conditions:default": None,
+        }),
+        hdrs = [":includes"],
```

Comment:
I'm not entirely sure, but having the same header files in different targets _might_ result in an ODR violation. It definitely could for `srcs`; I can never remember how `hdrs` gets processed differently, though. I'll ask around.